specification/riskiq/data-plane/Microsoft.Easm/preview/2023-03-01-preview/easm.json

```
@@ -0,0 +1,5635 @@
+          },
+          "type": "array"
+        },
+        "orgNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantContacts": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantPhones": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrarCreatedAt": {
+          "items": {
+            "$ref": "#/definitions/ObservedLong"
+          },
+          "type": "array"
+        },
+        "registrarNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrarUpdatedAt": {
+          "items": {
+            "$ref": "#/definitions/ObservedLong"
+          },
+          "type": "array"
+        },
+        "registries": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "sources": {
+          "items": {
+            "$ref": "#/definitions/Source"
+          },
+          "type": "array"
+        },
+        "technicalContacts": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalOrgs": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalPhones": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "AssetPageResponse": {
+      "description": "The page of assets that match the provided metric.",
+      "properties": {
+        "mark": {
+          "description": "The cursor mark to be used on the next request.  Not set if using paging.",
+          "type": "string"
+        },
+        "nextLink": {
+          "description": "The link to access the next page of results.  Not set if at the end of the result set.",
+          "type": "string"
+        },
+        "totalElements": {
+          "description": "The total number of items available in the full result set.",
+          "format": "int64",
+          "type": "integer"
+        },
+        "value": {
+          "description": "The items in the current page of results.",
+          "items": {
+            "$ref": "#/definitions/AssetResponse"
+          },
+          "type": "array"
+        }
+      },
```

The `value` property of a pageable response should be _required_:
```suggestion
      },
      "required": ["value"],
```